### PR TITLE
FEniCS adapter documentation

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
Mainly providing links to the repository and tutorial cases.

### CODE REVIEW

**Comment by BenjaminRodenberg** [comment]
We decided that the documentation should be provided directly on the website. Meaning: The content the links are currently pointing to should be moved to the website and the `README` of the repository should only contain minimal documentation.

File: pages/docs/adapters/fenics/adapter-fenics.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: The FEniCS adapter
+permalink: adapter-fenics.html
+keywords: adapter, fenics
+summary: "A python package providing a preCICE-adapter for the open source computing platform FEniCS."
+---
+
+## FEniCS adapter
+
+For details on installation and usage of the adapter, please refer to its github repository [`precice/fenics-adapter`](https://github.com/precice/fenics-adapter).
```
Comment (BenjaminRodenberg) [inline pages/docs/adapters/fenics/adapter-fenics.md:10]: Here, a section in **Installation** and **Testing** should be provided.

Comment (MakisH) [reply pages/docs/adapters/fenics/adapter-fenics.md:10]: What about the **Configuration** section?

**Review by MakisH — APPROVED** [review]
It builds nicely with the latest master, I don't see any reason to keep it back right now, despite the very minimal content.

**Pull request merged**